tools/etl/tg-jdbc-driver/README.md

```diff
@@ -4,13 +4,14 @@ The TigerGraph JDBC Driver is a Type 4 driver, converting JDBC calls directly in
 
 ## Versions compatibility
 
-| JDBC Version | TigerGraph Version | Java | Protocol | Query Result Format | New Features |
-| --- | --- | --- | --- | --- | --- |
-| 1.2.1 | 2.4.1~3.4 | 1.8 | Rest++ | ResultSet | Support interpreted queries and Spark partitioning |
-| 1.2.2 | 2.4.1~3.4 | 1.8 | Rest++ | ResultSet | Bug fix |
-| 1.2.3 | 2.4.1~3.4 | 1.8 | Rest++ | ResultSet | Bug fix |
-| 1.2.4 | 2.4.1~3.4 | 1.8 | Rest++ | ResultSet | Add vulnerability check plugin |
-| 1.2.5 | 3.5+ | 1.8 | Rest++ | ResultSet | Fix restpp authentication incompatibility |
+| JDBC Version | TigerGraph Version | Java | New Features |
+| --- | --- | --- | --- |
+| 1.2.1 | 2.4.1~3.4 | 1.8 | Support interpreted queries and Spark partitioning |
+| 1.2.2 | 2.4.1~3.4 | 1.8 | Bug fix |
+| 1.2.3 | 2.4.1~3.4 | 1.8 | Bug fix |
+| 1.2.4 | 2.4.1~3.4 | 1.8 | Add vulnerability check plugin |
+| 1.2.5 | 3.5+ | 1.8 | Fix restpp authentication incompatibility |
+| 1.2.6 | 2.4.1+ | 1.8 | Bug fix && support version selection |
```

Comment:
Please mention logging in the new features column, some customers are looking forward to it.
```
Support version selection, JUL and SLF4J
```